src/samplers/ldsampler.cpp

```diff
@@ -0,0 +1,135 @@
+#include <mitsuba/core/profiler.h>
+#include <mitsuba/core/properties.h>
+#include <mitsuba/core/spectrum.h>
+#include <mitsuba/core/qmc.h>
+#include <mitsuba/render/sampler.h>
+
+NAMESPACE_BEGIN(mitsuba)
+
+/**!
+
+.. _sampler-ldsampler:
+
+Low discrepancy sampler (:monosp:`ldsampler`)
+---------------------------------------------
+
+This plugin implements a simple hybrid sampler that combines aspects of a Quasi-Monte Carlo sequence
+with a pseudorandom number generator based on a technique proposed by Kollig and Keller
+:cite:`Kollig2002Efficient`. It is a good and fast general-purpose sample generator. There exists
+other QMC samplers that can generate even better distributed samples, but this comes at a higher
+cost in terms of performance. This plugin is based on Mitsuba 1's default sampler (also called
+:monosp:`ldsampler`).
+
+Roughly, the idea of this sampler is that all of the individual 2D sample dimensions are first
+filled using the same (0, 2)-sequence, which is then randomly scrambled and permuted using a
+shuffle network. The name of this plugin stems from the fact that (0, 2) sequences minimize the
+so-called star disrepancy, which is a quality criterion on their spatial distribution.
```

Comment:
Instead of "so-called", better to link to some page with more details (e.g. https://en.wikipedia.org/wiki/Low-discrepancy_sequence)

Reply:
While the (0,2) sequence clearly has a very low star discrepancy, I'm not sure if "minimizes" is the correct way of introducing this QMC sequence. (Probably only asymptotically? It's been a long time since I worked on these, and I don't remember 100%..). Anyways, would be good to have a mathematically sound description here, I'd look at how it is introduced in the relevant literature.